libs/ledger-live-common/src/wallet-api/types.ts

```diff
@@ -0,0 +1,64 @@
+import type { SignedOperation } from "@ledgerhq/types-live";
+import type {
+  CryptoCurrency,
+  TokenCurrency,
+} from "@ledgerhq/types-cryptoassets";
+
+export type {
+  Families as WalletAPIFamilies,
+  Account as WalletAPIAccount,
+  Currency as WalletAPICurrency,
+  Transaction as WalletAPITransaction,
+  CryptoCurrency as WalletAPICryptoCurrency,
+  ERC20TokenCurrency as WalletAPIERC20TokenCurrency,
+} from "@ledgerhq/wallet-api-core";
+
+export {
+  CurrencyType as WalletAPICurrencyType,
+  TokenStandard as WalletAPITokenStandard,
+} from "@ledgerhq/wallet-api-core";
+
+export type TranslatableString = {
+  en: string;
+  [locale: string]: string;
+};
+
+export type AppPlatform =
+  | "desktop" // == windows || mac || linux
+  | "mobile" // == android || ios
+  | "all";
+
+export type AppBranch = "stable" | "experimental" | "soon" | "debug";
+
+export type AppPermission = {
+  method: string;
+  params?: any;
+};
+
+// TODO update to the new manifest types from wallet-api when released
```

Comment:
Side note / idea on that: we could expose the AppManifest type from one of the `wallet-api` packages in order to not have to re-define it for each wallet (and be sure it is up to date / avoid duplication / have a single source of truth)

Reply:
Is it better exported from the client and/or server ?
IMO it's fine in another package like it is now, but as you said maybe it makes sense to export it from the client and server to avoid version shifts and simplify the usage

Reply:
I was intuitively thinking the `core` package